tests/test_hydra.py

```diff
@@ -649,32 +649,29 @@ def test_help(
 
 
 @mark.parametrize(
-    "script,overrides,expected",
+    "overrides,expected",
     [
         param(
-            "examples/tutorials/basic/your_first_hydra_app/1_simple_cli/my_app.py",
-            ["--info", "searchpath", "hydra.searchpath=['pkg://fakeconf']"],
```

Comment:
This is an app without hydra.searchpath in its config.
so it's testing hydra.searchpath in the overrides only.

I think you changed the meaning of the test.

Reply:
Hmmm in the validation we are  making sure that the overrides comes from the command-line so I don't think the meaning is changed. 

The reason I had to update the app is because we need the package to actually exist next to the app now. 
An alternative is to add an another primary  config next to `examples/advanced/config_search_path/my_app.py` with no hydra.searchpath defined in it,  I didn't do that because: this test is focused on the output of  `--info` and we are already checking to make sure the hydra.searchpath comes from command-line override. 

thoughts?

Reply:
Why did you use two different apps in the first place when creating it?

Reply:
it was not intentional :)